File: Assignment2.js
function findMax(arr){
    var max=arr[0];
    for(i=0;i<arr.length;i++)
    {
        if(arr[i]>max)
        {
            max=arr[i];
        }
    }
    return max;
}
var arr = [1,3,5,4,2,7,5,6,3,2,5];
console.log(findMax(arr));

File: assignment2.java
package com.company;
import java.util.*;
class assignment2{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter size of array: ");
        int n = scan.nextInt();
        int arr[] = new int[n];
        int max=0;
        int pos=0;
        System.out.println("Enter the value of array elements:");
        for(int i=0;i<n;i++)
        {
            arr[i]=scan.nextInt();
            if(max<arr[i]){
                max = arr[i];
                pos=i;
            }
        }
        System.out.println("Maximum Number: "+max);
        System.out.println("Position of First Occurrence: "+pos);
    }
}